Remove libtool check.

This enables configure to find either libtool or glibtool.
diff --git a/autogen.sh b/autogen.sh
index ec3378d2..24af5580 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -16,14 +16,6 @@ DIE=0
 	DIE=1
 }
 
-(libtoolize --version) < /dev/null > /dev/null 2>&1 || {
-	echo
-	echo "You must have libtool installed to compile libxslt."
-	echo "Download the appropriate package for your distribution,"
-	echo "or see http://www.gnu.org/software/libtool"
-	DIE=1
-}
-
 (automake --version) < /dev/null > /dev/null 2>&1 || {
 	echo
 	DIE=1
